user-interface/src/staff-assignment/modal/AssignAttorneyModal.tsx

```diff
@@ -260,7 +260,7 @@ function _AssignAttorneyModal(
             <label className="attorney-name">Attorney Name</label>
           </div>
           <div className="usa-table-container--scrollable" ref={tableContainer}>
-            <table className="attorney-list">
+            <table className="attorney-list" role="presentation">
```

Comment:
Pa11y was complaining about this element because the child elements contain semantic meaning. If this is a required part of these changed we can figure out adding it back